test/unittest/blas3/blas3_trsm_test.cpp

```diff
@@ -0,0 +1,114 @@
+
+#include "blas_test.hpp"
+
+template<typename scalar_t>
+using combination_t = std::tuple<int, int>;
+
+template <typename T>
+void transpose(T* A, size_t m, size_t n, size_t lda) {
```

Comment:
Isn't there already an existing function that can do this?

Reply:
No, but this is temporary and will be removed soon

Reply:
Removed